### ISSUE TITLE
Error signing or witnessing alonzo-era transactions

### ISSUE THREAD

**Issue by cardano-apexpool** [vacuumlabs/cardano-hw-cli#114]
When trying to sign or witness alonzo-era transactions (built with `cardano-cli transaction build` or, when using newer cardano-cli versions, also with `cardano-cli transaction build-raw` but wothout the `--mary-era` parameter), the transactions signing or witnessing fails with the following error message:

`Error: Invalid raw transaction CBOR`

The same error was in the previous version. I did many transactions since then, with different cardano-cli versions. I am using a Ledger Nano X, and I also used different firmware version of ledger, nothing changes. Last try was a few minutes ago, using the newest stable versions of everything (cardano-cli, cardano-hw-cli, ledger firmware).

**Comment by gitmachtl** [comment]
hw-cli and the cardano-app is not alonzo-transaction compatible yet. there will be an update soon.

**Comment by tdiesler** [comment]
You can easily downgrade to cardano-cli-1.31.0 like this ...

```
alias cardano-cli="docker run -it --rm \
  -v ~/cardano:/var/cardano/local \
  -v node-ipc:/opt/cardano/ipc \
  nessusio/cardano-node:1.31.0 cardano-cli"
```

for example if you need to change the pool config.

**Comment by gitmachtl** [comment]
Just make a `--mary-era` transaction, thats it. No need for a node rollback.

**Comment by cardano-apexpool** [comment]
> 

I made many updates since I posted that (last one yesterday), I always used `--build-raw` with `--mary-era`, with the lates cardano-node and cardano-cli versions available. This is not why I opened the issues. I opened the issue because I want it to be fixed, and I did not see other similar issues opened, so I did not know if it is a known problem. Then gitmachtl replied with another issue reporting the same problem.
From what I understand, a fix will probably be released in 2-3 months (because of the long reviews.
